docs/examples/huggingface/README.md

```diff
@@ -199,6 +199,46 @@ inference_request = {
 requests.post("http://localhost:8080/v2/models/transformer/infer", json=inference_request).json()
 ```
 
+### Masked Language Modeling
+
+We can also serve a masked language model. In the following example, we also build the `huggingface` runtime with the `-E japanese` flag to enable support for Japanese tokenizers.
```

Comment:
maybe provide a cmd how to build exactly?

Reply:
Hey @sakoush, provided a detailed command in my last commit and updated the header. Let me know if anything else needs to change!